### PR TITLE
Add gnomAD data source

### PR DESCRIPTION
**Pull request by haochenl**

### CODE REVIEW

File: docs/data-sources/gnomad-lof-json.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+}
+```
+| Field | Type  | Notes                                                                                                                        |
+|-------|-------|------------------------------------------------------------------------------------------------------------------------------|
+| pLi   | float | probability of being intolerant of a single loss-of-function variant (like haploinsufficient genes, observed ~ 0.1*expected) |
+| pNull | float | probability of being completely tolerant of loss of function variation (observed = expected)                                 |
+| pRec  | float | probability of being intolerant of two loss of function variants (like recessive genes, observed ~ 0.5*expected)             |
+| synZ  | float | corrected synonymous Z score                                                                                                 |
+| misZ  | float | corrected missense Z score                                                                                                   |
+| loeuf | float | loss of function observed/expected upper bound fraction (LOEUF)                                                              |
```
Comment (rajatshuvro) [inline docs/data-sources/gnomad-lof-json.md:18]: Should we capitalize the first letter of the sentences in Notes?

File: docs/data-sources/gnomad-small-variants-json.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+| femaleAc            | int    | allele count for female population. Integer.                                                     |
+| controlsAllAc       | int    | allele count for the controls subset. Integer.                                                   |
+| allAn               | int    | allele number for all populations. Non-zero integer.                                             |
+| maleAn              | int    | allele number for male population. Non-zero integer.                                             |
+| femaleAn            | int    | allele number for female population. Non-zero integer.                                           |
+| controlsAllAn       | int    | allele number for the controls subset. Non-zero integer.                                         |
+| allHc               | int    | count of homozygous individuals for all populations. Non-negative integer.                       |
+| maleHc              | int    | count of homozygous individuals for male population. Non-negative integer.                       |
+| femaleHc            | int    | count of homozygous individuals for female population. Non-negative integer.                     |
```
Comment (rajatshuvro) [inline docs/data-sources/gnomad-small-variants-json.md:68]: Capitalize first letter?

**Review by rajatshuvro — CHANGES_REQUESTED** [review]
Minor change about capitalizing first letter of 'Notes' column

File: docs/data-sources/gnomad-lof-json.md (hunk 1 above)
Comment (haochenl) [reply docs/data-sources/gnomad-lof-json.md:18]: Okay. I can do that.

File: docs/data-sources/gnomad-small-variants-json.md (hunk 2 above)
Comment (haochenl) [reply docs/data-sources/gnomad-small-variants-json.md:68]: @rajatshuvro Actually, when I looked at the "Notes" from the other tables, most of them start with lower case letters.

**Review by rajatshuvro — APPROVED** [review]

**Pull request merged**